### ISSUE TITLE
Experiment with surfacing AV landing page

### ISSUE THREAD

**Issue by bruce-usab** [atbcb/usab-uswds#1198]
From call 6-7, the AV landing page is not linked up.  We decided try three different approaches, and review next week 6-14.  Each of these should get its own branch, do **not** use beta.
- [ ] hero-light static version on homepage with link to AV page and webinar of the month @bruce-usab 
- [ ] cards on News landing page for AV and other emerging area @RSD-accessboard 
- [ ] carousel on home page for featured items @philbratta

**Comment by bruce-usab** [comment]
Prototype of [hero-light static version](https://federalist-e3fba26d-2806-4f02-bf0e-89c97cfba93c.app.cloud.gov/preview/atbcb/usab-uswds/sidekick/) is available.  Comments are most welcome!

**Comment by tc-usab** [comment]
Comments on “standby hero” homepage Mock up:
1. Centrally located, hard to miss when user lands on the page.
2. Color scheme is high contrast, easy to read.  Text is short and focused.
3. Background pictures (manual chair user on a public street) implies action and motion.  Good choice for the subject matter. Could be one of a series of similar pictures showing engagement of PwD in their respective environments.
4. Date on URL is helpful.  Demonstrates the "freshness" of the content.
5. Two subsidiary links are helpful - highlight continuously  updated content.  
Bottom line:  Good choice.  I suggest leaving it up for a test period (30-45 days) then post a short survey (?) asking for feedback from viewers "What do you think?"

**Comment by bruce-usab** [comment]
Per conversation 6/14, we are not going forward with the <q>hero-light</q> implementation.
